pyperf/run_pyperf

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+pip3_install()
+{
+	pip3 -q install $1
+}
+
+PYPERF_VERSION="1.0.4"
+export VIRTUAL_ENV_DISABLE_PROMPT=1
+python3 -m venv ${1}-env
+source ${1}-env/bin/activate
+python3 -m ensurepip --upgrade
```

Comment:
I think we should be installing pip via the python3-pip package

That is the most supported way I see across the distros.  Maybe if the command doesn't exist we run ensurepip.